src/index.css

```diff
@@ -48,6 +48,29 @@ body {
   padding: 8em 0em;
 }
 
+/* Predictions.js */
+.prediction-button {
```

Comment:
Am I missing something or did `prediction-button` not get added as a class on the Button it's supposed to apply to?

Reply:
Ok, snapshot has been updated this should hopefully be all good now?